### PR TITLE
Integrate SEP-2022 ASB patches to T-MR0-CELADON-MASTER

### PR DESCRIPTION
**Pull request by alavalas1**
Tracked-On: OAM-103675
Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>

### CODE REVIEW

File: aosp_diff/preliminary/build/make/0004-Update-Security-patch-level.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+diff --git a/core/version_defaults.mk b/core/version_defaults.mk
+index 65957c8859..5883925c73 100644
+--- a/core/version_defaults.mk
++++ b/core/version_defaults.mk
+@@ -103,7 +103,7 @@ ifndef PLATFORM_SECURITY_PATCH
+     #  It must be of the form "YYYY-MM-DD" on production devices.
+     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
+     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
+-    PLATFORM_SECURITY_PATCH := 2022-08-05
++    PLATFORM_SECURITY_PATCH := 2022-09-05
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/build/make/0004-Update-Security-patch-level.patch:23]: will it be 05 or 01?

Comment (alavalas1) [reply aosp_diff/preliminary/build/make/0004-Update-Security-patch-level.patch:23]: As per the T codebase, Aug month is 05

**Review by tanujtekriwal — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
LGTM

**Pull request merged**